docs/2-advanced/08-using-i18n-for-apps.md

```diff
@@ -6,7 +6,7 @@ None of the code below implies or requires the usage of UI5 Web Components, and
 The `@ui5/webcomponents-base` package allows the usage of `i18n` functionality not just for UI5 Web Components,
```

Comment:
can we add front matter at the beginning of the file
```
---
title: i18n for Apps
---
```

Reply:
sure